docs/getting_started/validate.md

```diff
@@ -0,0 +1,47 @@
+# Evaluate your GTFS feed’s quality
+
+A high-quality GTFS is complete, accurate, and up-to-date. This means that it represents how services are currently operating and provides as much information as possible. 
+
+## Complete data
+
+Quality GTFS includes important service details such as holiday and summer schedule changes, accurate stop locations, and names for routes and headsigns that match other rider-facing materials. Even if an agency works with a vendor to produce GTFS, it’s ultimately up to the agency to ensure that the information presented in print, on board, and online is consistent.
+
+## Accurate data
+
+Accurate data is essential for providing transit riders with a reliable and user-friendly transportation experience. Errors in the data can block a portion or the totality of a dataset from being used.
+
+## Up-to-date data
+
+Having out-of-date data is almost worse than having no feed at all. It's not enough to simply publish information—it has to match what the rider sees and experiences. Some of the largest transit agencies update their GTFS weekly, or even daily, but most agencies will need to update their GTFS every few months, or a few times a year when service changes. This includes things like new routes or stops, timetable changes, or updates to fare structure.
+
+Many agencies hire a vendor to create and manage their GTFS for them. Some vendors may be proactive in asking about service changes, but it’s important that agencies communicate with vendors about upcoming service changes. It’s possible to publish GTFS with service changes in advance, making sure the transition goes smoothly for everyone—agencies, vendors, trip planners, and riders!
+
+## Using official validators 
+
+There is a lack of consensus among stakeholders regarding what constitutes a valid dataset file. This results in data quality issues that are not addressed consistently across different organizations, as different stakeholders may have varying definitions and interpretations of what constitutes a violation. 
+
+The free and open-source canonical [GTFS Schedule validator](https://gtfs-validator.mobilitydata.org/)[^1] maintained by MobilityData ensures your GTFS data is compliant with the official GTFS Schedule Reference and Best Practices. It provides an easy-to-use validation report in an HTML format that can be opened in the browser and shared with other parties.
+
+### Usage
```

Comment:
I suggest we remove this sub-title. 
If we were to add sub-titles here, having "GTFS Schedule" and "GTFS Realtime" would probably be more appropriate.